bouncer/templates/base.html.jinja2

```diff
@@ -3,18 +3,22 @@
   <head>
     <meta charset=utf-8>
 
+    {% if can_reveal_metadata == True %}
```

Comment:
I probably messed this up in my other comments, but this should be `is True` (also with `None` and `False`), [here](https://stackoverflow.com/a/27276841)'s a StackOverflow answer providing some reasons on `is` vs `==`.